src/test/resources/ci/scripts/run.sh

```diff
@@ -172,7 +172,7 @@ main() {
 
     # Run the tests
     echo -e "\n$(${currentTime[@]}): INFO: Running tests..."
-    ./gradlew test
+    ./gradlew test -PuseLocal=$USE_LOCAL_PLUGIN
```

Comment:
Is the `useLocal` parameter required when invoking the `test` goal?

Reply:
I tried running tests without the `useLocal` parameter but did not achieve the desired results. I attempted to run tests using an `lsp4ij` branch with some code commented out. Ideally, the LTI tests should fail in this scenario. However, when I ran the tests using `./gradlew test` without specifying `useLocal`, the tests passed. The tests only failed when I included the `useLocal` parameter with `./gradlew test`. So useLocal is required

Reply:
I tried running tests without the `useLocal` parameter but did not achieve the desired results. I attempted to run tests using an `lsp4ij` branch with some code commented out. Ideally, the LTI tests should fail in this scenario. However, when I ran the tests using `./gradlew test` without specifying `useLocal`, the tests passed. The tests only failed when I included the `useLocal` parameter with `./gradlew test`. So useLocal is required

Reply:
I tried running tests without the `useLocal` parameter but did not achieve the desired results. I attempted to run tests using an `lsp4ij` branch with some code commented out. Ideally, the LTI tests should fail or show warnings since some code is missing out in lsp4ij in this scenario. However, when I ran the tests using `./gradlew test` without specifying `useLocal`, the tests passed and showed no warning. The tests only showed warning - `Caused by: com.intellij.diagnostic.PluginException: Missing extension point: com.redhat.devtools.lsp4ij.server in container Application (unit test) (internal) (headless) (command line) (RA allowed) (IR on EDT disabled) [Plugin: com.redhat.devtools.lsp4ij]` when I included the `useLocal` parameter with `./gradlew test`. So useLocal is required